app/src/main/java/org/openedx/app/data/storage/PreferencesManager.kt

```diff
@@ -41,6 +54,12 @@ class PreferencesManager(context: Context) : CorePreferences, ProfilePreferences
         }
         get() = getString(REFRESH_TOKEN)
 
+    override var accessTokenExpiresAt: Long
+        set(value) {
+            saveLong(EXPIRES_IN, value)
+        }
+        get() = getLong(EXPIRES_IN)
+
```

Comment:
It is not obvious that this stores time in seconds. 
This can lead to confusion.

Reply:
Handled by using milliseconds. 👍🏻